server/routes/journeys/bulk-alerts/upload-prisoner-list/schemas.ts

```diff
@@ -75,6 +75,18 @@ export const validateFileAndSubmitPrisonerList =
       const error = e as { text?: string }
       const errorRespData = JSON.parse(error?.text || '{}') as { userMessage?: string }
       if (errorRespData?.userMessage) {
+        const rowNumbers = [...errorRespData.userMessage.matchAll(/\d+/g)]
```

Comment:
This does work but it's pretty tightly coupled to the API message and potentially brittle. Would it be a lot of work to add a `includeHeaderRow` request boolean to the API and have it increment the row numbers if true?